samples/nrf9160/gps/src/supl_support.c

```diff
@@ -0,0 +1,154 @@
+#include <zephyr.h>
+#include <supl_session.h>
+#include <stdio.h>
+#include <logging/log.h>
+
+#define SUPL_SERVER "supl.google.com"
+
+static int            supl_fd;
+
+int open_supl_socket(void)
+{
+	int err;
+	int proto;
+	u16_t port;
+	struct nrf_addrinfo *addr;
+	struct nrf_addrinfo *info;
+
+	proto = NRF_IPPROTO_TCP;
+	port = nrf_htons(7276);
```

Comment:
The port number could have it's #define, just below `SUPL_SERVER`

Reply:
yes